packages/core/components/EmptyFileListMessage/index.tsx

```diff
@@ -35,21 +37,29 @@ export default function EmptyFileListMessage() {
                                     ))}
                                 </span>
                             )}
-                            {annotationHierarchyListItems.length > 0 && (
+                            {annotationHierarchy.length > 0 && (
                                 <span>
                                     {" "}
                                     with annotation
-                                    {annotationHierarchyListItems.length === 1 ? "" : "s "}
-                                    {map(annotationHierarchyListItems, (annotation, index) => (
-                                        <span key={annotation.id} title={annotation.description}>
-                                            {index > 0
-                                                ? index === annotationHierarchyListItems.length - 1
-                                                    ? " and "
-                                                    : ", "
-                                                : " "}
-                                            <b>{annotation.title}</b>
-                                        </span>
-                                    ))}
+                                    {annotationHierarchy.length === 1 ? "" : "s "}
```

Comment:
Eventually software will make the English language drift to orderly formation of plural words.  :)